dimod/discrete/cydiscrete_quadratic_model.pxd

```diff
@@ -40,6 +40,8 @@ cdef class cyDiscreteQuadraticModel:
     cdef vector[CaseIndex] case_starts_  # len(adj_) + 1
     cdef vector[vector[VarIndex]] adj_
 
+    cdef Bias offset_
```

Comment:
rather than making a private attribute with getter/setter, you can just do `cdef public Bias offset`

Reply:
Done